### ISSUE TITLE
Home page not loading after refresh

### ISSUE THREAD

**Issue by ShanikaEdiriweera** [ShanikaEdiriweera/temple-website#24]
Checkout dev branch

After the merge #23 something is broken. When reload the browser home page does not load. Should be something wrong with script loading

**Comment by singhcharanjot** [comment]
I am not able to reproduce this issue. Can you please add a screenshot and also please update as to which server are you using?

**Comment by tbsschroeder** [comment]
Your JS throws an error due to CORS policy ;-)

**Comment by singhcharanjot** [comment]
Yes the JS will throw an error if you are running the index page directly without any server and in such case the web address will start with "file:///....".  However when I am running the website using a php server it seems to work fine and the web address will start with "localhost:....."

If it's something else, please share and let me know :)

**Comment by singhcharanjot** [comment]
> Your JS throws an error due to CORS policy ;-)



> Your JS throws an error due to CORS policy ;-)

@tkrauthoff 
Please refer : 

https://stackoverflow.com/questions/10752055/cross-origin-requests-are-only-supported-for-http-error-when-loading-a-local

**Comment by ShanikaEdiriweera** [comment]
Sorry for the late response. @singhcharanjot and @tkrauthoff thanks for helping out. I am also using wamp server and localhost. 
The issue should be something else.

**Comment by singhcharanjot** [comment]
@ShanikaEdiriweera I tested the issue on with dev branch and was not able to replicate it. Can you please add a few screenshots of the issue and also output of the chrome console(press f12). if not we can plan a time and you can please share your screen and I will look into it

**Comment by ShanikaEdiriweera** [comment]
I deployed it to master branch. CHeck here https://shanikaediriweera.github.io/temple-website/index.html
Html is appearing and then dissapearing

**Comment by ShanikaEdiriweera** [comment]
@singhcharanjot  You can see this in the console. `[Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience.`

This happens because there are scripts inside the html loaded by `$("#homehtml").load("./pages/home.html")`. 

I searched on this. but could not find a proper solution.

**Comment by singhcharanjot** [comment]
Ok. I got the issue and will update you with a fix shortly
